src/components/publishing/header/feature_header.tsx

```diff
@@ -240,7 +243,7 @@ const FeatureHeaderContainer = Div.extend`
 `
 
 const sizeMeOptions = {
-  refreshRate: 64,
+  refreshRate: 500,
```

Comment:
What does this do in practice?

Reply:
From their docs:
  // The maximum frequency, in milliseconds, at which size changes should be
  // recalculated when changes in your Component's rendered size are being
  // detected. This should not be set to lower than 16.

This is half a second now, instead of 64 milliseconds. We currently have an issue where the browser is occasionally freezing components. There are (I believe) 2 reasons why...the first is that I implemented react-sizeme a little naively. It's being used multiple times within an article on the individual component as opposed to at MOST once or twice on the article. This would be a larger-scale refactor.

The second is this factor, where because we have multiple components with recalculations every 64 milliseconds, the browser can't keep up. Changing it to half a second gives us a little more breathing room (and actually doesn't break the browser anymore). I think we still need that larger refactor but for now this helps. 

Sorry this should have definitely been explained as part of the PR! Lmk if u have questions.

Reply:
Not sure if it applies to the second issue you were referring to, but you may also want to look at [`shouldComponentUpdate`](https://facebook.github.io/react/docs/react-component.html#shouldcomponentupdate).